.travis.yml

```diff
@@ -29,6 +29,7 @@ matrix:
     - env: LANGUAGE=react-native VERSION=1
     - env: LANGUAGE=ruby         VERSION=2.6
     - env: LANGUAGE=sonar        VERSION=3.3      GLIBC_VERSION=2.27-r0 SONARSCANNER_VERSION=3.3.0.1492
+    - env: LANGUAGE=scoutsuite   VERSION=5.1.0    PIP_VERSION=19.1.1 PIPENV_VERSION=2018.11.26 SCOUTSUITE_VERSION=5.1.0
```

Comment:
I think we should limit the precision of `VERSION` to the minor version (in this case `5.1`), like we do for other images. (well except for ansible it seems 😓)

That way, if someone references the image with `ekino/docker-buildbox:latest-scoutsuite5.1`, the image will have all new patches when pulled again. If not, users will have to upgrade from `ekino/docker-buildbox:latest-scoutsuite5.1.0` to `ekino/docker-buildbox:latest-scoutsuite5.1.1` etc.

It doesn't affect users using more precise / release tags of course, it's juste a choice we give for our images.

**Unless** scoutsuite has the habit of making breaking changes in patch versions of course.

What do you think @axelpavageau ?

Reply:
scoutsuite's pretty new (it's a new iteration of a previous project), and so far they've only release major versions.
However, personally, I like setting & using precise versions, but YMMV ...

Reply:
Good for me 👍

Reply:
> I think we should limit the precision of `VERSION` to the minor version (in this case `5.1`), like we do for other images. (well except for ansible it seems 😓)
> 
> That way, if someone references the image with `ekino/docker-buildbox:latest-scoutsuite5.1`, the image will have all new patches when pulled again. If not, users will have to upgrade from `ekino/docker-buildbox:latest-scoutsuite5.1.0` to `ekino/docker-buildbox:latest-scoutsuite5.1.1` etc.
> 
> It doesn't affect users using more precise / release tags of course, it's juste a choice we give for our images.
> 
> **Unless** scoutsuite has the habit of making breaking changes in patch versions of course.
> 
> What do you think @axelpavageau ?

That's why I suggested (in #172) to have one repo per image to follow usual docker image tag practice and allow better comprehension for users and allow us to specify more precise versions

Reply:
Yes and we'll be able to have different tagging strategies per repositories if we need to